api/openapi/my_api.yaml

```diff
@@ -520,6 +519,12 @@ components:
           example: "Age is used to calculate the creatinine clearance. Dosing is lower for geriatric patient and contraindicated for pediatric patients"
         certitude:
           type: integer
+          description: The certitude of the value, 0 not certain, 1 somewhat certain, 2 certain
+          example: 2
+        group:
+          type: string
+          description: "Provides the client with grouping patient variables into multiple groups"
+          example: "Body Type
```

Comment:
missing right quotation mark for example field, i.e., should be "Body Type"

Reply:
If we add "group" field into RequiredPatientVariable object as an optional field in pdspi-config, should we also add it into pds as well for consistency?